frontend/src/App.vue

```diff
@@ -25,6 +25,9 @@ export default {
       menu: { status, profiles } 
     }
   },
+  mounted() {
+    this.$store.dispatch('getAllStudents');
```

Comment:
can we use vuex helpers? like `mapActions` as much as possible we avoid using this.$store

Reply:
Nice suggestion! We have refactored to use vuex helpers. Thanks!